src/Uno.Templates/content/unoapp/MyExtensionsApp.1.Windows/Package.appxmanifest

```diff
@@ -34,6 +34,15 @@
         Description="MyExtensionsApp._1">
         <uap:SplashScreen BackgroundColor="#086AD1"/>
       </uap:VisualElements>
+<!--#if (useWebAuthentication)-->
```

Comment:
@dansiegel any idea why this condition wouldn't be evaluated? Is there something we need to do in order to get evaluations to be run on this file?

Reply:
Changed to //